circle.yml

```diff
@@ -0,0 +1,43 @@
+machine:
+    services:
+        - docker
+    node:
+        version: 5.5
+    pre:
+        - sudo curl -L -o /usr/bin/docker 'https://s3-external-1.amazonaws.com/circle-downloads/docker-1.9.1-circleci'
+        - sudo chmod 0755 /usr/bin/docker
+    environment:
+        IMAGE_NAME: couchbase
+        COUCHBASE_HOST: localhost
+        COUCHBASE_PORT: 8091
+        # Ottoman tests look for this.
+        CNCSTR: couchbase://localhost
+
+general:
+    build_dir: .
+    artifacts:
+        - ./coverage
+
+dependencies:
+    pre:
+        - docker version
+        - docker info
+        - npm prune && npm install
+    cache_directories:
+        - node_modules
+    override:
+        - docker info
+        - docker ps
+        - docker images
+        - docker pull mdavidallen/couchbase:latest
```

Comment:
NOTE!!!  This PR relies on a docker image that is built from this separate repo.

https://github.com/moxious/couchbase-docker-circleci

While it's OK with me if you use this image, in the medium term I would recommend cloning that repo, customizing a few variables for you, and then keeping your own copy so that changes to that repo don't inadvertently break ottoman testing.